lib/takeWhile.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * `takeWhile` take a slice of an array, elements will be taken 
+ * from the begining of the array until selector returns false
+ * 
+ * @params {Array} array from which values will be iterated
+ * @params {selector} delegate function to determine if we stop iterating
```

Comment:
I usually call these functions predicates. If that adds any clarity you are free to change it. If not don't worry about it.